src/main/java/seedu/tr4cker/model/task/Deadline.java

```diff
@@ -13,12 +13,14 @@
  */
 public class Deadline {
 
-
     public static final String MESSAGE_CONSTRAINTS =
-            "deadline times should only contain numbers, and it should follow the format yyyy-MM-dd HHmm";
+            "Deadline should be a valid date, and it should follow the format yyyy-MM-dd "
+                    + "or yyyy-MM-dd HHmm";
+    public static final String MESSAGE_FUTURE =
```

Comment:
could name it MESSAGE_FUTURE_CONSTRAINS to be clearer

Reply:
Yes, you're right.